GameEngine/GameEngine/Engine/BFC/BFCItem.hpp

```diff
@@ -17,8 +18,9 @@ namespace AGE
 		inline glm::vec4 getPosition() { return _position; }
 	private:
 		BFCCullableObject *_drawable = nullptr;  // 64
-		void        *_transformationInfos; //64
 		glm::vec4   _position; // 128
+	public:
+		void        *next; //64
 
```

Comment:
No, there is no padding
[
64   (8 * 8)
128 (16 * 8)
64   (8 * 8)

@paulbaron @herpec-j si vous vous y connaissez mieux n'hesitez pas a nous clarifier sur la question :smile_cat: 
]